spring-ws-core/src/main/java/org/springframework/ws/soap/addressing/version/AbstractAddressingVersion.java

```diff
@@ -200,7 +200,6 @@ public void addAddressingHeaders(SoapMessage message, MessageAddressingPropertie
 		if (map.getTo() != null) {
 			SoapHeaderElement to = header.addHeaderElement(getToName());
 			to.setText(map.getTo().toString());
-			to.setMustUnderstand(true);
```

Comment:
This is not mandatory by specification. Do you think we must provide a way to specify wether to set `mustUnderstand` attribute?

Reply:
Let's wait until someone asks.